internal/database/database_test.go

```diff
@@ -0,0 +1,69 @@
+package database
+
+import (
+	"github.com/autobrr/autobrr/internal/domain"
+	"github.com/autobrr/autobrr/internal/logger"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func getDbs() []string {
+	return []string{"sqlite", "postgres"}
+}
+
+func SetupDatabase(t *testing.T, dbType string) *DB {
```

Comment:
This does not need to be exported, and could maybe be renamed to `setupDatabseForTest` to be more descriptive?

Reply:
:+1: